### PR TITLE
Ms teams bot id fix

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-22426
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-22122

## Description
Fixed the bug where the BOT_ID was not sent to the API.

## Failing UT
image

### CODE REVIEW

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 1]
```diff
@@ -1305,7 +1305,7 @@ def is_bot_in_chat(chat_id: str) -> bool:
     url_suffix = f"v1.0/chats/{chat_id}/installedApps"
     res = http_request('GET', urljoin(GRAPH_BASE_URL, url_suffix),
                        params={"$expand": "teamsApp,teamsAppDefinition",
-                               "$filter": "teamsApp/externalId eq '{BOT_ID}'"})
+                               "$filter": f"teamsApp/externalId eq '{BOT_ID}'"})
```
Comment (Shellyber) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:1308]: nice

**Review by dantavori — APPROVED** [review]

**Review by Shellyber — COMMENTED** [review]
Nice.
Just need to add release notes and bump the pack version

**Review by merit-maita — APPROVED** [review]

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py [hunk 2]
```diff
@@ -210,7 +221,9 @@ def process_incident_create_message(demisto_user: dict, message: str) -> str:
             incidents: Union[dict, list] = json.loads(incidents_json.replace('“', '"').replace('”', '"'))
             if not isinstance(incidents, list):
                 incidents = [incidents]
-            created_incident = create_incidents(demisto_user, incidents)
+
+            add_req_data_to_incidents(incidents, request_body)  # type: ignore[arg-type]
```
Comment (merit-maita) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:225]: i don't recommend sending the whole request_body as it is, it includes confidential data

**Review by merit-maita — CHANGES_REQUESTED** [review]
looks good overall, just the one comment about sending the req_body

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py (hunk 2 above)
Comment (ShahafBenYakir) [reply Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py:225]: Good to know, let's sync offline on which values are confidential.

**Comment by ShahafBenYakir** [comment]
@merit-maita the rawJson after sanitization:
image

Also added info to docs.

**Review by merit-maita — APPROVED** [review]

**Pull request merged**